server_admin/topics/export-import.adoc

```diff
@@ -33,14 +33,14 @@ To export into unencrypted directory you can use:
 bin/standalone.sh -Dkeycloak.migration.action=export
 -Dkeycloak.migration.provider=dir -Dkeycloak.migration.dir=<DIR TO EXPORT TO>
 ----
-And similarly for import just use `-Dkeycloak.migration.action=import` instead of `export` .
```

Comment:
This sentence should be moved down since it is a description for import command.